_misc/sanity_utils.py

```diff
@@ -112,9 +112,6 @@ def find_files(
             _remove_ignored_directories(path, dirs, ignored_dirs, ignored_path_regexps)
             for filename in files:
                 filepath = posixpath.join(path, filename)
-                if filename == "generated_resources.txt":
-                    _process_generated_resources(path, filepath, generated_resources)
-                    continue
```

Comment:
The _process_generated_resources function is now probably unused and should be removed too.